packages/back-end/src/scim/groups/listGroups.ts

```diff
@@ -0,0 +1,72 @@
+import { parse, filter } from "scim2-parse-filter";
+import { getTeamsForOrganization } from "../../models/TeamModel";
+import { expandOrgMembers } from "../../services/organizations";
+import { createApiRequestHandler } from "../../util/handler";
+import { COUNT_DEFAULT, START_INDEX_DEFAULT } from "../users/listUsers";
+import {
+  ScimGroup,
+  ScimListRequest,
+  ScimListResponse,
+} from "../../../types/scim";
+
+export const listGroups = createApiRequestHandler()(
```

Comment:
I noticed that on the group endpoints, we're still using the `createApiRequestHandler` whereas in the `user` endpoints, we're not. 

Do we need the `createApiRequestHandler`?

Reply:
We don't need it, but it makes typing the responses easier since it takes care of wrapping the object returned in the appropriate response and catches any zod schema issues (we should probably add that for SCIM requests). We'd probably need a special SCIM request handler if we go this route though because errors are not returned in the way SCIM expects them in this handler.